os/mm/mm_heap/mm_check_heap_corruption.c

```diff
@@ -100,11 +101,9 @@ static void dump_node(struct mm_allocnode_s *node, node_type_t type)
 		return;
 	}
 
-	if (type == TYPE_CORRUPTED) {
-		mfdbg("CORRUPTED NODE: addr = 0x%08x size = %u preceding size = %u\n", node, node->size, MM_PREV_NODE_SIZE(node));
-	} else if (type == TYPE_OVERFLOWED) {
-		mfdbg("OVERFLOWED NODE: addr = 0x%08x size = %u type = %c\n", node, node->size, IS_ALLOCATED_NODE(node) ? 'A' : 'F');
-	}
+	bool is_allocated = IS_ALLOCATED_NODE(node);
+	mfdbg("%s: addr = 0x%08x type = %c size = %u preceding size = %u\n", node_type, node, is_allocated ? 'A' : 'F', node->size, MM_PREV_NODE_SIZE(node));
```

Comment:
`is_allocated` is used only once from here, so we don't need to use local variable.